R/model_builder.R

```diff
@@ -85,6 +85,10 @@ build_kmeans.kv <- function(df,
   model_column <- avoid_conflict(grouped_column, "model")
   source_column <- avoid_conflict(grouped_column, "source.data")
 
+  if(row_col %in% grouped_column){
+    stop(paste0(row_col, " is a gruoping column. You can use ungroup() to solve this."))
```

Comment:
Typo: "gruoping".
"You can use..." does not sound very well as error message. Maybe something like following?
XXX is a grouping column. ungroup() may be necessary before this operation.